exercises/roman-numerals/instructions.md

```diff
@@ -0,0 +1,12 @@
+# Introduction
+
+Your task is to convert a number from Arabic numerals (0-9) to Roman numerals.
```

Comment:
We show an example of the digits available for Arabic numerals, but do not give the same courtesy to the Roman numerals.  I think that without the Arabic range shown, most people will know, it is the Roman numerals that would more likely need an example here.

Reply:
Roman numerals have just been introduced 'above' in the `introduction.md`. The explanation includes a table.

I imagine for many people used to only Arabic numerals these numerals are functionally nameless, or referred to as 'digits', 'numbers', 'number signs', &c.